### ISSUE TITLE
Can't build AuraBot

### ISSUE THREAD

**Issue by Shizuke** [Josko/aura-bot#69]
Hello i'm having 3 errors while trying to build AuraBot:
1_The Referenced Project '..\StormLib\StormLib.vcxproj' does not exist.
2_Error MSB8036: The Windows SDK version 8.1 was not found.
3_Error MSB8036: The Windows SDK version 10.0.14393.0 was not found.

I never used Visual Studio so i don't know if what i did was wrong.
Here is what i did because there is no guide to compile it:
Downloaded full visual studio 2017
Downloaded AuraBot as ZIP
Double clicked on aura.sln
Clicked on Compile -> Compile Solution

**Comment by Shizuke** [comment]
After i installed windows sdk it gives these errors:
1_ The Referenced Project '..\StormLib\StormLib.vcxproj' does not exist.
2_ LNK1181 Can't open file 'StormLibRUS.lib'

**Comment by JFGHT** [comment]
> Hello i'm having 3 errors while trying to build AuraBot:
1_The Referenced Project '..\StormLib\StormLib.vcxproj' does not exist.
2_Error MSB8036: The Windows SDK version 8.1 was not found.
3_Error MSB8036: The Windows SDK version 10.0.14393.0 was not found.

> I never used Visual Studio so i don't know if what i did was wrong.
Here is what i did because there is no guide to compile it:
Downloaded full visual studio 2017
Downloaded AuraBot as ZIP
Double clicked on aura.sln
Clicked on Compile -> Compile Solution

This happened to me yesterday.

What I did to fix it was:

- Installing VS C++ 2015 libraries. You can do it by running the installer, select the current installed version, click modify and in the right corner, on the bottom, you'll have it.

- Open the solution. Then on the solutions explorer, right click on the solution -> redestiny solution -> select your current SDK -> accept.

IT should work now.

**Comment by Josko** [comment]
Exactly. Due to being compatible with vs2015, the vs2017 project has the older toolset selected. Ideally the project should have a vcxproj file for each version but that is tedious as I'm rarely on Windows.

Like @JFGHT explain nicely, at the bottom of the `Desktop development with C++` category:

![image](https://user-images.githubusercontent.com/13797/38736305-e5078c32-3f2b-11e8-8c6d-62d4c29e12f7.png)

**Comment by NikiSkaarup** [comment]
you can right click each project in the solution and change the Windows 10 SDK to the one you have installed and it should work with the latest one available, I've been able to successfully build and run it with it, results may vary ofc
